tstest/integration/vms/vms_test.go

```diff
@@ -415,20 +446,39 @@ func TestVMIntegrationEndToEnd(t *testing.T) {
 		for port := range ipMap {
 			port := port
 			t.Run(port, func(t *testing.T) {
-				config := &ssh.ClientConfig{
-					User:            "ts",
-					Auth:            []ssh.AuthMethod{ssh.PublicKeys(signer), ssh.Password("hunter2")},
-					HostKeyCallback: ssh.InsecureIgnoreHostKey(),
+				t.Parallel()
+				time.Sleep(5 * time.Second)
```

Comment:
why? sleeps are never the right answer (both too long and too short). it needs a comment at least.

Reply:
I thought I purged that as part of the delay loop. Sorry!